lib/src/component/dom_components.dart

```diff
@@ -910,3 +911,27 @@ abstract class Dom {
   /// optionally backed by a specified map.
   static SvgProps vkern([Map backingMap]) => SvgProps(react.vkern, backingMap);
 }
+
+class $DomPropsImpl extends DomProps with $DomPropsMixin {
+  $DomPropsImpl(componentFactory, [Map props]) : super(componentFactory, props);
+}
+
+DomPropsConfig _$domPropsConfig = DomPropsConfig(
+  propsFactory: PropsFactory(
+    map: (factory, props) => $DomPropsImpl(factory, props),
+  )
+);
+
+class DomPropsConfig<T extends builder_helpers.UiProps> {
+  PropsFactory propsFactory;
+
+  DomPropsConfig({this.propsFactory});
+}
+
+class PropsFactory<T extends DomProps> {
```

Comment:
Is this supposed to be the same [PropsFactory that will be used for function components](https://github.com/Workiva/over_react/compare/new_boilerplate_spike/cleanup#diff-535a825af0d04e24ba42b9978f959082R296), or something different?